scipy-notebook/Dockerfile

```diff
@@ -57,8 +62,9 @@ RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
     'numba=0.23*' \
     'bokeh=0.11*' \
     'h5py=2.5*' \
-    'pyzmq' \
-    && conda clean -tipsy
+    'pyzmq' && \
+    conda remove --force qt pyqt && \
```

Comment:
I think this need the prefix too ( `-p $CONDA_DIR/envs/python2` ). Also, I forgot to add `--quiet --yes`.